manager_datacenter.go

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package main
+
+import (
+	"encoding/json"
+
+	"github.com/fatih/color"
+)
+
+// Datacenter ...
+type Datacenter struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+	Type string `json:"type"`
+}
+
+// CreateVcloudDatacenter ...
+func (m *Manager) CreateVcloudDatacenter(token string, name string, rtype string, user string, password string, url string, network string, vseURL string) (string, error) {
+	payload := []byte(`{"name": "` + name + `", "type":"` + rtype + `", "region": "LON-001", "username":"` + user + `", "password":"` + password + `", "external_network":"` + network + `", "vcloud_url":"` + url + `", "vse_url":"` + vseURL + `"}`)
+	body, _, err := m.doRequest("/api/datacenters/", "POST", payload, token, "")
+	if err != nil {
+		return body, err
+	}
+	color.Green("SUCCESS: Datacenter " + name + " created")
+	return body, err
+}
+
+// CreateAWSDatacenter ...
+func (m *Manager) CreateAWSDatacenter(token string, name string, rtype string, awstoken string, awssecret string) (string, error) {
+	payload := []byte(`{"name": "` + name + `", "type":"` + rtype + `", "region": "LON-001", "username":"` + name + `", "token":"` + awstoken + `", "secret":"` + awssecret + `"}`)
```

Comment:
Not sure why it's hardcoded for vcloud but, region shouldn't be hardcoded for AWS datacenters. The documentation specifies it needs to be provided by the end user with:
```
$ ernest datacenter create aws --region region --token token --secret secret --fake my_datacenter
```